segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/extendeds3/ExtendedS3Storage.java

```diff
@@ -279,9 +286,10 @@ private SegmentProperties syncCreate(String streamSegmentName) throws StreamSegm
                 new Grant(new CanonicalUser(config.getAccessKey(), config.getAccessKey()),
                         Permission.FULL_CONTROL)
         });
-
         request.setAcl(acl);
 
+        //TODO: As described above, this should be replaced with setIfNoneMatch("*") once the issue with extended S3 API is fixed.
```

Comment:
A TODO requires an issue associated to it:

https://github.com/pravega/pravega/wiki/Contributing#todos